vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
             "_links": [
                 {
                     "rel": "self",
-                    "href": "/vdbs/portfolio"
+                    "href": "/vdbs/Portfolio"
                 },
                 {
                     "rel": "content",
-                    "href": "/vdb/portfolio"
+                    "href": "/vdb/Portfolio"
```

Comment:
If a client does a GET on /vdbs then everything underneath will be returned from the server, eg. /vdbs/Portfolio, /vdbs/Portfolio/blah, /vdbs/Portolio/blah/blah. Therefore, if the content is placed beneath /vdbs then that will come back too and we may as well add the content straight under each vdb in /vdbs.
Now a vdb's content is quite verbose and thus a good few KB, eg. VdbsContent.json is 157KB for only 3 vdbs. I can imagine that mounting up the vdbs in the workspace is going to increase that size by quite a lot. As the point of displaying the collection of vdbs in the UI, there is little point in having the content to hand, especially given there are potentially 10, 20, 50 vdbs. Only when a single vdb is selected is the content now important. Thus, at that point, the content should be fetched from the server and displayed accordingly.
Thus, a separate resource providing the content ensures that its sizeable data amount is kept away from the much smaller metadata that describes the vdbs collection. Also, requesting 1 vdb at a time means that a 30KB json object is downloaded rather than a 50-vdb json object (1.5M).